cognite/client/data_classes/model_hosting/schedules.py

```diff
@@ -7,9 +7,8 @@ class Schedule(CogniteResource):
     """A representation of a Schedule in the model hosting environment.
 
     Args:
-        id (int): Id of the schedule.
-        model_id (int): The id of the model associated with this schedule.
         name (str): Name of the schedule.
```

Comment:
For consistency we should maybe use `schedule_name` here as well?

Reply:
Here as well, since this is a resource object, it needs to match the json returned from Nostromo.